stetho/src/main/java/com/facebook/stetho/inspector/elements/android/ViewDescriptor.java

```diff
@@ -72,6 +124,125 @@ public View getViewForHighlighting(Object element) {
     return (View)element;
   }
 
+  @Override
+  protected void onGetStyles(View element, Accumulator<StyleInfo> styles) {
+    for (Method method : viewGetters) {
+      boolean caught = false;
+      try {
+        ViewDebug.ExportedProperty annotation = getExportedAnnotation(method);
+        Object returnedValue = method.invoke(element);
+        String cssName = convertGetterNameToCssName(method.getName());
+        String cssValue = String.valueOf(returnedValue);
+        boolean isValueDefault = false;
+
+        // special case id to translate it into a human-readable string
+        if (method.getName().equals("getId")) {
+          cssValue = getIdAttribute(element);
+
+          if (cssValue == null) {
+            cssValue = "(none)";
+          }
+        } else {
+          if (method.getReturnType() == int.class) {
+            int intValue = (int) returnedValue;
+
+            if (canIntBeMappedToString(annotation)) {
+              cssValue += " (" + mapIntToStringUsingAnnotations(intValue, annotation) + ")";
+              isValueDefault = false;
+            } else {
+              isValueDefault |= isNumberDefault(intValue, annotation);
+            }
+          } else if (method.getReturnType() == float.class) {
+            isValueDefault |= isNumberDefault((Number)returnedValue, annotation);
+          }
+        }
+
+        styles.store(new StyleInfo(cssName, cssValue, isValueDefault));
+      } catch (IllegalAccessException e) {
+        caught = true;
+      } catch (InvocationTargetException e) {
+        caught = true;
+      } finally {
+        if (caught) {
+          LogUtil.w("failed to get the style of element id=" +
+              getIdAttribute(element));
+        }
+      }
+    }
+  }
+
+  private static boolean canIntBeMappedToString(
+      ViewDebug.ExportedProperty annotation) {
+    return annotation != null
+           && annotation.mapping().length > 0;
```

Comment:
style: if we're putting && on the next line, convention is to pull it under the return like so:

```
return annotation != null
    && annotation.mapping.length() > 0;
```

'return' is 6 characters, and so 4 space indent + 2 space operator results in things being lined up